webcompat/webhooks/model.py

```diff
@@ -234,19 +242,20 @@ def get_public_issue_number(self):
         return url
 
     def classify(self):
-        """Make a request to bugbug and label the issue.
+        """Make a request to bugbug and milestone the issue.
 
-        Gets issue classification from bugbug and labels
+        Gets issue classification from bugbug and milestones
         the issue if probability is high
         """
         data = get_issue_classification(self.number)
         needsdiagnosis_false = data.get('class')
         proba = data.get('prob')
 
         if needsdiagnosis_false and proba and proba[1] > THRESHOLD:
-            payload = {'labels': ['bugbug-probability-high']}
```

Comment:
I have decided to not use this label in private repo because in addition to the milestone it seems unnecessary